geoh5py/shared/conversion/grid2d.py

```diff
@@ -53,15 +53,6 @@ def grid_to_tag(input_entity: Grid2D) -> dict:
         :param input_entity: the Grid2D object to convert.
         :return tag: the tag dictionary.
         """
-        if not isinstance(input_entity.origin, np.ndarray):
```

Comment:
not possible anymore with the default instantiation?

Reply:
Correct